swagger.yaml

```diff
@@ -532,6 +532,58 @@ paths:
         "default":
           description: Software entry not found
 
+  /users:
+    get:
+      operationId: listUsers
+      summary: Get list of keycloak users.
+      description: Get list of keycloak users.
+      tags:
+        - Users
+      responses:
+        "200":
+          description: Get list of keycloak users
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/User"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
```

Comment:
Error schema does not currently exists, it's good to use it like this but it needs to be created first.

Reply:
Also pasting your spec in the swagger editor would highlight issues like this